### PR TITLE
Libraries area pod updates for March 2023, plus some fabricbot maintenance

### PR DESCRIPTION
**Pull request by jeffhandley**
Each of the commits applies one change to make the chunks easier to review.

**Please do not merge on my behalf. This PR, dotnet-api-docs/pull/8924, and dotnet/machinelearning/pull/6607 all need to be merged at about the same time as renaming the referenced GitHub projects.**

1. Regenerate the fabricbot.json file using dotnet/fabricbot-config/pull/71 to reflect @smasher164, @dakersnar, and @bartonjs no longer being members of the area pods
2. Tag @dotnet/area-system-componentmodel-dataannotations instead of @jeffhandley directly for https://github.com/dotnet/runtime/labels/area-System.ComponentModel.DataAnnotations
3. Add @marek-safar as a mentionee for https://github.com/dotnet/runtime/labels/linkable-framework (which was the desired effect of #82228
4. Update `area-owners.md` to reflect area pod changes and regenerate the `area-owners.json` file by parsing `area-owners.md` using [area-owners-md-to-json.ps1](https://raw.githubusercontent.com/jeffhandley/dotnet-area-pods/main/area-owners-md-to-json.ps1)
5. Add @bartonjs as a direct mentionee for areas where he will remain consulted
6. Sort all of the areas alphabetically within the issue routing portion of the config (in preparation of filling in missing areas)
7. Add recently added areas that were not added into the issue routing config
8. Switch https://github.com/dotnet/runtime/labels/area-System.Speech to use a team mention instead of @ericstj and @jeffhandley directly

### CODE REVIEW

File: docs/area-owners.md [hunk 1]
```diff
@@ -89,9 +89,9 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
-| area-System.Formats.Tar                        | @jeffhandley         | @dotnet/area-system-formats-tar                     | Consultants: @bartonjs @jozkee                                                                                                                                                                                                                                                            |
+| area-System.Formats.Asn1                       | @jeffhandley         | @dotnet/area-system-formats-asn1                    | Consultants: @bartonjs                                                                                                                                                                                                                                                                    |
```
[the remark this answers is not in the document]
Comment (jeffhandley) [reply docs/area-owners.md:92]: We can still reach out to @bartonjs for these areas as needed for domain expertise. He's asked to keep getting tagged on issues/PRs for these areas too, similarly to how @vcsjones gets tagged for several areas.

Comment (vcsjones) [inline docs/area-owners.md:92]: If you want, feel free to mark me available for consulting for `area-System.Formats.Asn1` and `area-System.Formats.Cbor`.

File: docs/area-owners.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
-| area-System.Security                           | @jeffhandley         | @dotnet/area-system-security                        |                                                                                                                                                                                                                                                                                           |
+| area-System.Security                           | @jeffhandley         | @dotnet/area-system-security                        | Consultants: @bartonjs @GrabYourPitchforks @SteveHarter @vcsjones                                                                                                                                                                                                                         |
```
Comment (ericstj) [inline docs/area-owners.md:123]: Should we also include wfurt?

**Review by ericstj — APPROVED** [review]
Libraries changes LGTM.

**Pull request merged**